### PR TITLE
Update libjpeg meta-package to deliver libjpeg-turbo

### PR DESCRIPTION
**Pull request by Toasterson**

### CODE REVIEW

File: components/meta-packages/libjpeg/Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		libjpeg
-COMPONENT_VERSION=	6.0.2
+COMPONENT_VERSION=	8.3.2
 COMPONENT_SUMMARY=	Joint Photographic Experts Group library
```
Comment (AndWac) [inline components/meta-packages/libjpeg/Makefile:20]: Shouldn't we enhance the summary?

Comment (Toasterson) [reply components/meta-packages/libjpeg/Makefile:20]: Sure, I commited what I could come up with. Open to suggestions though.

**Comment by mtelka** [comment]
Since nothing in OpenIndiana depends on this package I suggest to obsolete it.  One less useless meta-package is always good.

**Comment by Toasterson** [comment]
Agreed. Moving into new PR as I killed my git history in the process :)

**Pull request closed without merging**